_posts/2024-07-30-survey-announcement.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "MDAnalysis Community Survey 2024"
+---
+
+We are excited to announce the launch of our [MDAnalysis Community Survey 2024](https://docs.google.com/forms/d/e/1FAIpQLSeTn6zIm-LIry7aRJ0IF_r_B4m4E0ddUEVIHxXGD-H3n-nOiQ/viewform?usp=sf_link)! Do you or your institution use, develop, contribute to, or sponsor MDAnalysis? Have you attended MDAnalysis workshops or user group meetings? Have you heard about MDAnalysis, but are still learning how to get started or looking for more ways to get involved? If any of these things sound like you, we want to hear from you as part of our valued community! Your feedback is crucial in helping us understand the diverse needs and experiences within our community and guiding our efforts to make MDAnalysis an even more inclusive and welcoming community.
+
+## Purpose of the Survey
+
+This survey aims to gather your insights and feedback on various aspects of MDAnalysis, especially about your experiences as a community member and your engagement with the MDAnalysis project. 
+
+## What will the data be used for?
+
+The information from this survey will help us foster an inclusive and supportive community by:
+
+- Determining preferred MDAnalysis communication channels and developing a content strategy to keep the community informed.
+- Informing workshop and event planning efforts to reach diverse audiences based in various time zones across the globe.
+- Ensuring MDAnalysis resources are findable and accessible.
+
+## Confidentiality and Anonymity
+
+We value your privacy so your responses will be kept **confidential and anonymous** and only shared with MDAnalysis survey administrators. Demographic information will only be reported in an aggregated format to generate high-level insights.
+
+## How to Participate
+
+We invite all members of the MDAnalysis community to participate in the survey. We greatly appreciate your time answering a few questions (around 10 minutes) before **August 31st, 2024**. Please take a moment to share your thoughts and experiences with us by clicking the link, [MDAnalysis Community Survey 2024](https://docs.google.com/forms/d/e/1FAIpQLSeTn6zIm-LIry7aRJ0IF_r_B4m4E0ddUEVIHxXGD-H3n-nOiQ/viewform?usp=sf_link), to access the survey.
+
+Thank you so much for being a part of the MDAnalysis community. We cannot wait to hear from you!😀
+
+## Acknowledgment
+
+This survey was developed as part of an [Outreachy project](https://www.outreachy.org/alums/2024-05/).
```

Comment:
```suggestion
This survey was developed as part of an [Outreachy project](https://www.mdanalysis.org/2024/05/27/gsoc-outreachy-students/#adetutu-oluwasanmi-develop-a-communications-strategy-for-a-growing-mdanalysis-user-and-contributor-base).
```

Reply:
I wonder if it makes more sense to link to @adetutudeborah's write-up on the MDAnalysis blog intro. This still links to to the Outreachy website, but also includes information on accessing @adetutudeborah's blog in case folks are interested in learning more about the project and development of the survey.